example/ios/App/App/capacitor.config.json

```diff
@@ -5,13 +5,5 @@
 	"bundledWebRuntime": false,
 	"ios": {
 		"scheme": "App"
-	},
```

Comment:
got applied when I ran `npx cap sync`

Reply:
it gets re-added if I `npm install` in `./example` from this branch

Reply:
thanks for pointing that out, that got removed but my project still built ok so I thought I was nothing much.

when I "npm install" I had to bundle install from the ios/app folder but then it gets re-added on the `npx cap sync`